server/views/applications/check-your-answers.njk

```diff
@@ -0,0 +1,134 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% extends "../partials/layout.njk" %}
+
+{% block pageTitle %}
+ Approved Premises - Check your answers
+{% endblock %}
+
+{% block content %}
+  {{ govukBackLink({
+      text: "Back",
+      href: paths.applications.show({ id: application.id })
+  }) }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-width-container">
+      <h1 class="govuk-heading-xl">
+        Check your answers
+      </h1>
+
+      <div class="main-box">
+        <div class="box-title govuk-heading-m">
+          Person Details
+        </div>
+        <div class="box-content" data-cy-check-your-answers-section="person-details">
+          {{ govukSummaryList({
+          rows: ([
+            {
+              key: {
+                text: "Names"
+              },
+              value: {
+                text: application.person.name
+              }
+            },
+            {
+              key: {
+                text: "CRN"
+              },
+              value: {
+                text: application.person.crn
+              }
+            },
+            {
+              key: {
+                text: "Date of Birth"
+              },
+              value: {
+                text: formatDate(application.person.dateOfBirth, { format: 'short' })
+              }
+            },
+            {
+              key: {
+                text: "NOMS Number"
+              },
+              value: {
+                text: application.person.nomsNumber
+              }
+            },
+            {
+              key: {
+                text: "Nationality"
+              },
+              value: {
+                text: application.person.nationality
+              }
+            },
+            {
+              key: {
+                text: "Religion or belief"
```

Comment:
Would capitalise 'B' for consistency with "Gender Identity", "Date of Birth", etc